openapi/p4pa-process-executions.openapi.yaml

```
@@ -1,31 +1,98 @@
-openapi: 3.1.0
+openapi: 3.0.1
 info:
   title: p4pa-process-executions
   description: Api and Models
-  version: "1.0.0"
+  version: 0.0.1
+servers:
+  - url: http://localhost:8080
+    description: Generated server url
 paths:
-  /api/v1/greet:
-    get:
-      summary: "Hello World"
-      description: "Sample endpoint that return greetings"
+  /:
```

let's explicit the entity that is being created
```suggestion
  ingestionflowfiles/:
```